detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -203,34 +219,30 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
         nInteractionLengths += slice_thickness/(2.*slice_material.intLength());
         thickness_sum += slice_thickness/2;
                 
-	if ( compSlice.isSensitive() )  {
```

Comment:
Found it. I don't see all changes in this box. That makes it difficult to see what is still missing.